app/src/i18n/translations.ts

```diff
@@ -0,0 +1,107 @@
+        deleteEntryMsg: 'Eintrag löschen',
+        saveTxt: 'Speichern',
+        backTxt: 'Zurück'
+      },
+      TableView: {
+        viewTooltip: 'Anzeigen',
+        zoomToGeometryTooltip: 'Auf Geometrie zoomen',
+        editTooltip: 'Bearbeiten',
+        deleteTooltip: 'Eintrag löschen',
+        seeItemViewText: 'Siehe Detailansicht',
+        createEntryText: 'Eintrag erstellen',
+      }
+    }
+  },
+  en: {
+    translation: {
+      ErrorPage: {
+        checkUrlMsg: 'Please check if you entered the correct address or try again later.',
+        checkParametersMsg: 'Please provide the `formId` and `view` parameters in the URL.',
+        checkViewParameterMsg: 'Please check the `view` parameter in the URL.',
+        headingText: 'Oops, something went wrong!',
+        viewNotSupportedMsg: 'The requested view "{{view}}" is not available.' +
+          'Please use either "item" or "table" as the view.'
+      },
+      ErrorCodes: {
+        unknownError: 'Unknown error',
+        unknownDatabaseError: 'Unknown error during database query',
+        CONFIG_KEY_IS_EMPTY: 'Configuration key is empty',
+        DB_JOIN_ITEMS_NOT_FOUND_FOR_CONFIG_KEYS: 'No join items found for configuration keys',
+        FORM_CONFIG_NOT_FOUND: 'The requested form with ID {{formId}} was not found',
+        FORM_ID_MISSING: 'Form ID parameter is missing',
+        INTERNAL_SERVER_ERROR: 'Internal server error',
+        INVALID_FILE: 'Invalid or missing file',
+        INVALID_FILTER: 'Invalid filter. Please check the filter criteria.',
+        INVALID_ORDER: 'Invalid sort order. Please use "asc" or "desc".',
+        INVALID_ORDER_BY: 'Invalid sort criteria. Please use valid column names.',
+        INVALID_PAGE: 'Invalid page. Please provide a valid page number.',
+        ITEM_CREATION_FAILED: 'Failed to create item',
+        ITEM_ID_MISSING: 'Item ID parameter is missing',
+        JOIN_TABLE_DATA_CREATION_FAILED: 'Failed to create join table data',
+        JOIN_TABLE_DATA_DELETION_FAILED: 'Failed to delete join table data',
+        JOIN_TABLE_NOT_FOUND: 'Join table not found',
+        JOIN_TABLE_PROP_NOT_FOUND: 'Join table property not found',
+        JOIN_TABLE_RELATIONSHIP_NOT_SUPPORTED: 'Join table relationship not supported',
+        LOOKUP_TABLE_PROPERTY_NOT_FOUND: 'Lookup table property not found',
+        MISSING_LOOKUP_COLUMNS: 'Missing lookup columns',
+        MISSING_LOOKUP_TABLES: 'Missing lookup tables',
+        DATABASE: {
+          XX000: 'Geometry error: Invalid/empty geometry cannot be saved.'
+        }
+      },
+      ItemView: {
+        deleteEntryMsg: 'Delete entry',
+        saveTxt: 'Save',
+        backTxt: 'Back'
+      },
+      TableView: {
+        viewTooltip: 'View',
+        zoomToGeometryTooltip: 'Zoom to geometry',
+        editTooltip: 'Edit',
+        deleteTooltip: 'Delete entry',
+        seeItemViewText: 'See detail view',
+        createEntryText: 'Create entry'
+      }
+    }
+  }
+};
```

Comment:
Minor: Some error messages end with a `.` (dot) others don't. Maybe we can harmonize this.

Reply:
👍🏼 , see [63a6aad](https://github.com/formcapture/form-backend/pull/164/commits/63a6aaddfc4e0d129eb6b76b17d35e90af3f5edf)